pkg/model/datasource/componentroot.yaml

```
@@ -0,0 +1,4 @@
+groups:
+  - coremodel
+owners:
+  - "@grafana/backend-platform"
```

Compgen's syntax expects this to not include the `@` symbol.
```suggestion
  - "grafana/backend-platform"
```